libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/local/video/MediaVideoView.kt

```diff
@@ -238,7 +235,8 @@ private fun ExoPlayerMediaVideoView(
 
     OnLifecycleEvent { _, event ->
         when (event) {
-            Lifecycle.Event.ON_RESUME -> exoPlayer.play()
+            Lifecycle.Event.ON_CREATE -> exoPlayer.addListener(playerListener)
```

Comment:
How it wasn't necessary before?

Reply:
The listener was set in a LaunchedEffect block (previous version line 132, see the diff)

Reply:
Ok, I didn't see it!